client/lib/mapMarkers.coffee

```diff
@@ -0,0 +1,32 @@
+Meteor.mapHelpers = {
```

Comment:
It would be preferable to use export/require modules than add things to the Meteor object. Here's an example of how that works:
https://github.com/ecohealthalliance/spa/blob/master/imports/data/feeds.coffee#L40
https://github.com/ecohealthalliance/spa/blob/master/client/templates/trend_map.coffee#L1